draft-edm-protocol-greasing.md

```diff
@@ -156,6 +156,35 @@ cause a new class of interoperability failure. Therefore a label such as
 "reserved for greasing" may help to avoid the confusion.
 
 
+# Deployment Considerations and Incentives for Greasing
+
+Greasing can be used as a tool to improve the active use of existing protocol
```

Comment:
Is "improve the active use" correct here? ... it seems like this could improve/exercise extensibility or something?